rfq/manager.go

```diff
@@ -949,6 +950,10 @@ func (m *Manager) getAssetGroupKey(ctx context.Context,
 	// Perform the DB query.
 	group, err := m.cfg.GroupLookup.QueryAssetGroup(ctx, id)
 	if err != nil {
+		if strings.Contains(err.Error(), "asset group is unknown") {
```

Comment:
Can we do a more specific error check here so we don't need to rely on string matching?

Reply:
Also when would this fail? Instances where we haven't synced the group key yet?

Reply:
We should resolve that circular dependency. Can be done by applying https://github.com/lightninglabs/taproot-assets/commit/fad16bf8171c4650dd60518d090d20aa2548561e (in the upstream branch `tinygo`).